templates/bind/named.conf.local.j2

```diff
@@ -17,18 +23,23 @@ masters {{ master.name }} {
 {% endif %}
 {% if bind9_masters_extra|default() %}
 {%   for master in bind9_masters_extra %}
-masters {{ master.name }} {
+masters "{{ master.name }}" {
 {%     for addr in master.addresses %}
     {{ addr }};
 {%     endfor %}
 };
 {%   endfor %}
 {% endif %}
+{% if bind9_acl is defined %}
```

Comment:
`bind9_acl` is missing in defaults.yml

Reply:
Ok, good point. However, we should also mention `bind9_acl` in defaults.yml and/or in README.md

Reply:
> `bind9_acl` is missing in defaults.yml
As we start with `if bind9_acl is defined` there is no need to set a default value, just to document (what was not yet finished)